sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -100,8 +106,82 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       *
       * @group tasks
       */
+    @deprecated("Use npmInstall or npmInstallDependencies", "0.13.2")
```

Comment:
I think we should make it clearer that the behavior of `npmUpdate` is now implemented by the `npmInstall` task.

Reply:
E.g.: 

> Use `npmInstall` (or `npmInstallDependencies` if you don’t need .js resources from the classpath)

?